ta1/output.py

```diff
@@ -0,0 +1,232 @@
+
+    name: str = Field(
+        ..., description="Name of the page extraction object", example=["Legend"]
+    )
+    ocr_text: str = Field(..., description="OCR text of the page extraction")
+    color_estimation: Optional[str]
+    bounds: Polygon = Field(
+        ..., description="Bounds of the page extraction, in pixel coordinates"
+    )
+    model: Optional[ExtractionIdentifier]
+
+
+class GroundControlPoint(BaseModel):
+    """Ground control point"""
+
+    id: int = Field(..., description="Internal ID")
+    map_geom: Point = Field(..., description="Point geometry")
+    px_geom: Point = Field(..., description="Point geometry")
+
+
+class ProjectionMeta(BaseModel):
+    """Information about the map projection. Projection information should also be applied
+    to the map image and output vector data (if using GeoPackage output format)."""
+
+    gcps: list[GroundControlPoint] = Field(..., description="Ground control points")
+    projection: WKT = Field(..., description="Map projection information")
+
+
+class ModelRun(BaseModel):
+    """Information about a model run."""
+
+    pipeline_name: str = Field(..., description="Model name")
+    version: str = Field(..., description="Model version")
+    timestamp: str = Field(..., description="Time of model run")
+    batch_id: Optional[str] = Field(..., description="Batch ID")
+    image_size: list[int] = Field(..., description="Pixel size of the map image")
+    confidence: list[ConfidenceEstimation]
+    boxes: list[PageExtraction]
+
+
+class CrossSection(BaseModel):
+    """Information about a geological cross section (lines of section + images).
+
+    NOTE: This would be nice to have but isn't required (especially for the initial target).
+    """
+
+    id: int = Field(..., description="Internal ID")
+    label: str = Field(..., description="Cross section label")
+    line_of_section: Line = Field(..., description="Geographic line of section")
+    image: Image = Field(..., description="Image of the cross section")
+
+
+class Map(BaseModel):
+    """Basic information about the extracted map."""
+
+    name: str = Field(..., description="Map name")
+    source_url: str = Field(
+        ..., description="URL of the map source (e.g., NGMDB information page)"
+    )
+    # A centralized map image store would simplify the work of debugging and re-running models.
+    image_url: str = Field(
+        ...,
+        description="URL of the map image, as a web-accessible, cloud-optimized GeoTIFF",
+    )
+    authors: str = Field(..., description="Map authors")
+    publisher: str = Field(..., description="Map publisher")
+    year: int = Field(..., description="Map publication year")
+    organization: str = Field(..., description="Map organization")
+    scale: str = Field(..., description="Map scale")
+    bounds: Polygon = Field(..., description="Map geographic bounds")
+
+    features: MapFeatureExtractions
+    cross_sections: list[CrossSection]
```

Comment:
Cross-sections are a "stretch" target for Macrostrat — it'd be nice to extract the images/page bounding boxes and "lines of section" on the map for these, but it's not required at the beginning.

Reply:
Commenting this out and holding for potential future extensions might be appropriate.